ChangeLog

```diff
@@ -1,3 +1,7 @@
+* 0.15.1
+    * Added support of 'warnings' to `status` command
```

Comment:
And by the way, maybe it's time to rename this file into one with proper extension? `ChangeLog.md`?